### PR TITLE
Section 2

### PR DESCRIPTION
**Pull request by tsytsykvitaliy**
Finished Section 2. Please take a look and let me know if you have any suggestions or does everything look ok. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
...
+- **Length**: 86  
+_Number of bytes in this packet_
+- **Info**: Standard query 0x332e A www.offensive-security.com  
+_Summary of the information in the highest layer protocol_
 
+This is the easiest way to see the most common information about the packets. However, there is much more that can be uncovered.
+
+#### Digging Deeper
+
+Glancing at the _Packet Listing_ is always nice, but it's rarely enough. If we want to learn more about the packet, we should click on it and explore the _Packet Details_. We have already discussed what information goes in each one of the layers (drop down rows) in the previous section so it will be easier to analyze our specific DNS packet, knowing that information. We will start from the top of the list with the physical layer, and work our way to the bottom - the "highest" protocol.
```
Comment (OwlWatcher) [inline README.md:159]: "previous section, so it will be" (add comma)

Comment (OwlWatcher) [reply README.md:159]: I think you can get rid of the comma before "knowing that information."

Comment (OwlWatcher) [reply README.md:159]: And no need for a comma after "physical layer, and work"

File: README.md [hunk 2]
```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
...
+
+##### Physical Layer
+
+![](physical.png)  
+_Screenshot of the physical layer information we have about the DNS packet._  
+
+There is plenty of rows to look at and there are lots of resources out there that break them all down, but right now we are just going to focus on the most important information:
```
Comment (OwlWatcher) [inline README.md:166]: "There is plenty of rows"  I think it should be "There are plenty of rows"

Comment (OwlWatcher) [reply README.md:166]: Or just make it They're

File: README.md [hunk 3]
```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
...
+
+- **Arrival Time**: Feb 3, 2010 16:26:52.129870000 Central Standard Time  
+_Time of the packet arrival at the destination_
+- **Epoch Time**: 1265236012.129870000  
+_Also called UNIX time, this is a number of seconds since January 1, 1970_
```
Comment (OwlWatcher) [inline README.md:171]: I think it should be "this is the number of seconds"

File: README.md [hunk 4]
```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
...
+- **Queries**: www.offensive-security.com: type A, class IN  
+_All requests included in this packet. In our case, we only have one_  
+
+As we can see, there is plenty of information about the packets that we can access with Wireshark. All the things we were able to tell about the DNS query packet #3 is visible by just clicking on the packet in the _Packet Listing_ window and paying close attention to the fields. However, this is a single packet. What if we want to search through hundreds of packets to find something specific? This is where our next topic comes in - sorting and filtering.
+
+### Sorting and Filtering
+
+#### Sorting Columns
+
+Let's look back at where we started: the _Packet Listing_ window. We saw all the standard columns like **Number** or **Protocol**. But what if we want to sort captured packets by something else? For example the **Destination Port**?
```
Comment (OwlWatcher) [inline README.md:249]: I think it should be "For example, the Destination Port" (Comma after for example)

File: README.md (hunk 1 above)
Comment (deviningers) [reply README.md:159]: "knowing that information" sounds a little repetitive due to you already saying "We have already discussed..."

File: README.md [hunk 5]
```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
...
+_Screenshot of the Command Menu and the Display Filter_  
+
+_Display Filter_ doesn't quite work the way `Ctrl + F` does on the web pages, so it's important to understand the correct way to build the expressions.
+
+Let's look at the basic syntax.  
+
+- To only display packets containing a specific protocol - just type in that protocol.  
+_For example, `http`_  
+
+- To filter them out further - type in that protocol, period, and further filterign options.
```
Comment (deviningers) [inline README.md:269]: filtering* instead of filterign

**Review by deviningers — APPROVED** [review]
Looks really good, I liked the in-depth explanations, the pictures, and the learn more link at the bottom.

**Comment by TomDanner16** [comment]
Holy buckets! This is some awesome info!!!! Looks EXCELLENT to me!

**Review by TomDanner16 — APPROVED** [review]

**Pull request merged**